spec/factories/items.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :item do
+    name "Gift1"
```

Comment:
Just a thought. You've defined `Gift1` name. Were I debugging some tests, and saw such name I would assume other gifts have names like `Gift2`, `Gift3` and so on. While we don't require a unique name, it would nice to generate them unique.